flexmeasures/data/models/planning/tests/test_solver.py

```diff
@@ -353,14 +379,24 @@ def test_building_solver_day_2(
         assert soc >= max(soc_min, battery.get_attribute("min_soc_in_mwh"))
         assert soc <= battery.get_attribute("max_soc_in_mwh")
 
-    # Check whether the resulting soc schedule follows our expectations for 8 expensive, 8 cheap and 8 expensive hours
+    # Check whether the resulting soc schedule follows our expectations for:
+    # 1) 8 expensive, 8 cheap and 8 expensive hours
+    # 2) 8 net-consumption, 8 net-production and 8 net-consumption hours
+
+    # 1) The battery sold out at the end of its planning horizon
```

Comment:
These three assumptions should be in the order of time progression, and also put a comment above ech:

- Result after 8 hours
- Result after second 8 hour-interval
- Result at end of day